docs/user_guide/HOWTO/Exclude_Repositories.rst

```diff
@@ -4,23 +4,46 @@ HOWTO Exclude YUM Repositories
 ==============================
 
 By default, SIMP applies updates from all available repositories on a
-nightly basis. This ensures that bug fixes and security updates are
-applied to all systems without minute management in Puppet manifests.
+nightly basis via the ``simp::yum`` class.
+
+The purpose of running a ``yum update`` on a nightly basis is to ensure that
+all critical updates are applied to the systems in a timely manner.
+Additionally, this places the package management burden on the repositories
```

Comment:
This is a very important point, and one of the operating assumptions that SIMP makes that was previously undocumented.